runtime/test-fw/src/pic_tests.rs

```diff
@@ -0,0 +1,169 @@
+// Licensed under the Apache-2.0 license
+
+#![no_main]
+#![no_std]
+
+use caliptra_drivers::{cprintln, Pic, SocIfc};
+use caliptra_registers::{el2_pic_ctrl::El2PicCtrl, soc_ifc::SocIfcReg};
+use caliptra_test_harness::test_suite;
+
+#[cfg(target_arch = "riscv32")]
+core::arch::global_asm!(include_str!("ext_intr.S"));
```

Comment:
Can you get away with include_str!("../../extr_intr.S") and delete this file? I don't like copy-and-paste.

Reply:
Ah sure. Older iterations had similar code to NMI to vector into rust eventually. That didn't work so I had to revert it. With the new insights from the meeting I might give that another try.

Reply:
I added it back since it works with a few nop cycles after enabling global interrupts.